ProcessMaker/Http/Controllers/Api/UserController.php

```diff
@@ -109,11 +111,12 @@ public function index(Request $request)
      */
     public function store(Request $request)
     {
-        $request->validate(User::rules());
         $user = new User();
-        $user->fill($request->input());
+        $user->fill($request->json()->all());
+        //validate model
+        $this->validateModel($user, User::rules($user));
```

Comment:
Why are we validating in a different way? This it not the way we want to do it.

Reply:
This way he assured me to validate only the fields sent.

for example. change status of user  only send field "status=INACTIVE" if I checking with rules with "validate request" I have the error "Field is required" in (username, etc...)